<launch>
	<node name ="overlay_2D_manual" pkg="nir_overlay" type="overlay_2D">
		<param name = "roi_l" value = "200"/>
		<param name = "roi_r" value = "200"/>
		<param name = "roi_u" value = "150"/>
		<param name = "roi_b" value = "150"/>
	</node>
	<node name = "gui" pkg ="rqt_gui" type ="rqt_gui"/>
</launch>